tests/functional/d/duplicate/duplicate_argument_name.py

```diff
@@ -1,14 +1,23 @@
 """Check for duplicate function arguments."""
 
+# pylint: disable=missing-docstring, line-too-long
+
 
 def foo1(_, _): # [duplicate-argument-name]
-    """Function with duplicate argument name."""
+    ...
 
 def foo2(_abc, *, _abc): # [duplicate-argument-name]
-    """Function with duplicate argument name."""
+    ...
 
 def foo3(_, _=3): # [duplicate-argument-name]
-    """Function with duplicate argument name."""
+    ...
 
 def foo4(_, *, _): # [duplicate-argument-name]
-    """Function with duplicate argument name."""
+    ...
+
+def foo5(_, *_, _=3): # [duplicate-argument-name, duplicate-argument-name]
+    ...
+
+# +1: [duplicate-argument-name, duplicate-argument-name, duplicate-argument-name, duplicate-argument-name]
+def foo6(_, /, _, *_, _="_", **_):
```

Comment:
```suggestion
def foo6(_, /, _, *_, _="a", **_):
```
Underscore is not the most readable variable name to use imo, but good enough. Also, for clarity it's probably clearer if there's something else than an underscore unless really needed to create the message?

If I understand correctly we could have this too ?
```suggestion
def foo6(_, /, _, *a, _="my string", **b):
```

Reply:
As-is the message is emitted 4 times; so following the suggestion will emit it 2 times instead.
Having said that, I don't disagree it's difficult to read - I went a bit mad with underscore :D

I could also split this into separate functions - just to make the illustration a bit easier to digest?

Reply:
Let me know if the latest commit is reasonable to improve this.
(also maybe the underscore was originally used to supress the unused-argument message :))

Reply:
Ho yeah, I misunderstood what it was doing, and now it's clear 😄